File: Java/Car.java
package CursoPOOUber.Java;



class Car {
    Integer id;
    String license;
    Account driver;
    private Integer passenger;

    public Car(String license, Account driver){
        this.license = license;
        this.driver = driver;
        
    }

    public Integer getPasseger(){
        return passenger;
    }
    
    public void setPassegenger(Integer passenger){
        this.passenger = passenger;
    }
    

    void printDataCar(){
        System.out.println("licence: "+ license + " Nombre driver: "+ driver.name + " passenger: "+passegenger);
    }
}

File: JS/Car.js
/*function Car(license, driver){
    this.id;
    this.license = license;
    this.driver = driver;
    this.passenger;
}

Cars.prototype.printDataCar = function(){
    console.log(this.driver)
    console.log(this.driver.name)
    console.log(this.driver.document)
}
*/
class Car {
    constructor (license, driver){
        this.license = license
        this.driver = driver
        this.id
        this.passenger
    }

    printDataCar = () => {
        console.log(this.driver)
        console.log(this.driver.name)
        console.log(this.driver.document)
    }
}